cloudformation/ad/ad-integration.yaml

```diff
@@ -21,14 +21,13 @@ Parameters:
     MaxLength: 64
     AllowedPattern: (?=^.{8,64}$)((?=.*\d)(?=.*[A-Z])(?=.*[a-z])|(?=.*\d)(?=.*[^A-Za-z0-9\s])(?=.*[a-z])|(?=.*[^A-Za-z0-9\s])(?=.*[A-Z])(?=.*[a-z])|(?=.*\d)(?=.*[A-Z])(?=.*[^A-Za-z0-9\s]))^.*
     NoEcho: true
-  UserName:
-    Description: Cluster user that is created in the Active Directory.
+  UserNames:
+    Description: Comma separated cluster users that are created in the Active Directory.
     Type: String
     Default: user000
     MinLength: 3
-    MaxLength: 64
   UserPassword:
-    Description: Cluster user Password.
+    Description: Cluster user Password for all users.
```

Comment:
*minor* I'd rephrase "Password for all the users specified in 'Users' parameter.